docs/source/tools.rst

```diff
@@ -1,13 +1,59 @@
 Tools
 =====
 
-Extra tools shipped as CLI with banana.
+Extra tools shipped as |CLI| with banana.
 
-GeneratePDF
------------
+genpdf
+------
 
-We provide also a console script called ``generatete_pdfs`` that is able to generate a custom PDF
-set in the `lhapdf` format,
-just providing a suitable function or constant value for each PID.
+We provide also a console script called ``genpdf`` that is able to generate and install a custom |PDF|
+set in the `lhapdf` format. In particular, the command ``genpdf install [NAME]`` simply install the |PDF| called [NAME]
+in the lhapdf folder and the ``genpdf generate [NAME]`` command generates the custom |PDF| and saves it as [NAME] in
+the current directory. Notice that the argument [NAME] is the only mandatory one.
```

Comment:
Well, specifing no labels is also quite useless :wink:

Reply:
same as @giacomomagni  added here https://github.com/N3PDF/banana/pull/16#pullrequestreview-810993796

Reply:
@andreab1997 I'd deny that option and throw an error - maybe you can adjust click?

Reply:
yes of course